### ISSUE TITLE
assertTrue does not work with "set -e"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kward/shunit2#37]
```
The below script crashes but would works fine if you remove '-e'

    #! /bin/sh -e

    testTrue()
    {
      assertTrue 0
    }

    . shunit2


I'm using shunit2 v2.1.6-1 from ubuntu raring

```

Original issue reported on code.google.com by `ert...@gmail.com` on 21 Jul 2013 at 9:28

**Comment by alexharv074** [comment]
@kward this is quite a gotcha. Is this issue just waiting on someone to rebase the above patch on the latest shunit2 in master?

**Comment by williamdes** [comment]
Fixed by 6f6a6423f5d6f873d649c5210a0f1254c0aeaafa
